unified-runtime/source/adapters/offload/program.cpp

```diff
@@ -70,85 +71,6 @@ ur_result_t ProgramCreateCudaWorkaround(ur_context_handle_t, const uint8_t *,
 }
 #endif
 
-// https://clang.llvm.org/docs/ClangOffloadBundler.html#bundled-binary-file-layout
```

Comment:
Was there a reason this comment/link was removed?

Reply:
Nah just sloppy on my part. Thanks! Fixed.